diff --git a/src/sd_driver/SDIO/rp2040_sdio.pio b/src/sd_driver/SDIO/rp2040_sdio.pio
index d40dcfb..40899b4 100755
--- a/src/sd_driver/SDIO/rp2040_sdio.pio
+++ b/src/sd_driver/SDIO/rp2040_sdio.pio
@@ -36,7 +36,7 @@
 ;   PINCTRL_IN_BASE configuration, modulo 32.
 ; This is used as a WAIT index, and must be between 4 and 31.
 ; (Offsets 0-3 are D0, D1, D2, and D3.)
-.define PUBLIC SDIO_CLK_PIN_D0_OFFSET 30    ; (-2 in mod32 arithmetic)
+.define PUBLIC SDIO_CLK_PIN_D0_OFFSET 5    ; (-2 in mod32 arithmetic)
 
 ; State machine 0 is used to:
 ; - generate continuous clock on SDIO_CLK
